app/views/search/_view_all_link.html.erb

```diff
@@ -2,9 +2,14 @@
   <div class="results-more">
   <% if @results['total'] > 0 %>
     <% return unless @results['total'] > @per_box %>
+    <% if Flipflop.enabled?(:local_browse) && @results['local_view_more'] %>
```

Comment:
Does it make sense to pull this logic out into a feature setting area? As we add more pieces that get toggled (I'm thinking of local browse for view all, subject, and author links at this point), we'll have logic scattered across several files.